### PR TITLE
Improve doc

### PR DESCRIPTION
**Pull request by fuzzythecat**
### Summary
This PR improves documentations on data iteration orders. Related to [this](https://github.com/keras-team/keras/pull/12035). 

### PR Overview

- [n] This PR requires new unit tests [y/n] (make sure tests are included)
- [n] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [y] This PR is backwards compatible [y/n]
- [n] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

**Comment by rragundez** [comment]
The comment added is wrong for `DataFrameIterator`. I'll check for `DirectoryIterator`.

**Comment by fuzzythecat** [comment]
My bad. Removed the comment.

File: keras_preprocessing/image/image_data_generator.py [hunk 1]
```diff
@@ -596,6 +597,7 @@ def flow_from_dataframe(self,
                 `model.predict_generator()`, `model.evaluate_generator()`, etc.).
             batch_size: size of the batches of data (default: 32).
             shuffle: whether to shuffle the data (default: True)
+                If set to False, sorts the data in alphanumeric order.
```
Comment (rragundez) [inline keras_preprocessing/image/image_data_generator.py]: Please remove as this returns a `DataFrameIterator`

Comment (fuzzythecat) [reply keras_preprocessing/image/image_data_generator.py]: Done

**Pull request merged**